docs/README.md

```diff
@@ -0,0 +1,115 @@
+
+Read [this](https://github.com/cleanflight/cleanflight/blob/master/docs/Introduction.md) for a small introduction about what CleanFlight and how the community works.
+
+## Hardware
+NOTE: Flight Controllers are typically equipped with accelerometers. These devices are sensitive to shocks. When the device is not yet installed  to an aircraft, it has very little mass by itself. If you drop or bump the controller, a big force will be applied on its accelerometers, which could potentially damage them. Bottom line: Handle the board very carefully until it's installed on an aircraft!
+
+For an overview of the hardware CleanFlight (hereby CF) can run on, see [Boards.md](https://github.com/cleanflight/cleanflight/blob/master/docs/Boards.md). For information about specific boards, see the files "Boards - xxxx.md", where xxxx is the board's name, in [the project's documents](https://github.com/cleanflight/cleanflight/tree/master/docs).
+
+* Assuming that you have a flight controller board (hereby FC) in hand, you should first read through the manual that it came with. You can skip the details about software setup, as we'll cover that here.
+
+* Decide how you'll connect your receiver by reading [this document](https://github.com/cleanflight/cleanflight/blob/master/docs/Rx.md), and how many pins you need on the outputs (to connect ESCs and servos) by reading about [Mixers](https://github.com/cleanflight/cleanflight/blob/master/docs/Mixer.md). 
+
+* If you're interested in monitoring your flight battery with CF, see [Battery Monitoring](https://github.com/cleanflight/cleanflight/blob/master/docs/Battery.md).
+
+* You may want audible feedback from your copter so skim through [Buzzer](https://github.com/cleanflight/cleanflight/blob/master/docs/Buzzer.md) and mark the pins that will be used.
+
+* Do you want your RC Receiver's RSSI to be sent to the board? [This](https://github.com/cleanflight/cleanflight/blob/master/docs/Rssi.md) explains how. You may or may not need to make an additional connection from your Receiver to the FC.
+* Would you like to try using a GPS unit to get your aircraft to Loiter or Return-To-Launch? Take a look [here](https://github.com/cleanflight/cleanflight/blob/master/docs/Gps.md) and [here](https://github.com/cleanflight/cleanflight/blob/master/docs/Gps_-_Tested_Hardware.md).
+
+* Now that you know what features you are going to use, and which pins you need, you can go ahead and solder them to your board, if they are not soldered already. Soldering only the pins required for the application may save weight and contribute to a neater looking setup, but if you need to use a new feature later you may have to unmount the board from the craft and solder missing pins, so plan accordingly.
+
+* If you are going to use [Oneshot125](https://github.com/cleanflight/cleanflight/blob/master/docs/Oneshot.md), flash your ESCs with the latest stable firmware and enable Damped Light in their settings, if it's supported.
```

Comment:
I would add here that flashing ESC firmware would mostly be for BLHeli ESCs. KISS ESCs are one of the major implementers of OneShot and they can't be reflashed (firmware not available).